src/minicraft/item/ToolItem.java

```diff
@@ -52,7 +64,7 @@ public boolean isDepleted() {
 	
 	/** You can attack mobs with tools. */
 	public boolean canAttack() {
-		return true;
+		return type != ToolType.Shear;

```

Comment:
I mean... technically speaking, getting hit with a pair of shears would still hurt lol

Reply:
So you don't kill the sheep